### PR TITLE
Add custom Redis cache handler to site

### PR DESCRIPTION
**Pull request by nsams**
Adds a custom cache handler for caching in Redis based on https://github.com/vivid-planet/comet/pull/2596 (and https://github.com/vivid-planet/comet/pull/2634).

## Open TODOs/questions

-   [x] Should we enable Redis locally? (like in Demo)

### CODE REVIEW

**Review by johnnyomair — COMMENTED** [review]
Please fix the lint workflow

**Comment by johnnyomair** [comment]
> Should we enable Redis locally? (like in Demo)

It would be closer to the real environment, so maybe it's a good idea?

**Comment by nsams** [comment]
> It would be closer to the real environment, so maybe it's a good idea?

Maybe. But it adds more complexity the the dev setup. What do you prefer?

> However, every dev can enable it using .env.local.

I also didn't add redis to docker-compose (only in demo) - so currently it is not _that_ easy to activate.

**Comment by johnnyomair** [comment]
> What do you prefer?

It should be easy to debug the Redis locally if there are ever any problems related to caching in the Redis. However, having the Redis running doesn't add any benefit, since we only have one site "pod" locally. So maybe add it to the docker-compose.yml but put deactivate it by default?

**Comment by johnnyomair** [comment]
> how? commenting the service out?

Maybe by combining [docker compose profiles](https://stackoverflow.com/a/78189900) and [COMPOSE_PROFILES](https://docs.docker.com/compose/how-tos/profiles/#start-specific-profiles)? Or two docker-compose files. Commenting out is fine as well I guess

**Comment by nsams** [comment]
profiles might be a solution but seem rather complicated and something new to learn (for devs using it) -  I added it commented out for now

File: .env [hunk 1]
```diff
@@ -64,3 +64,9 @@ API_BASIC_AUTH_SYSTEM_USER_PASSWORD=$BASIC_AUTH_SYSTEM_USER_PASSWORD
 JAEGER_UI_PORT=16686
 JAEGER_OLTP_PORT=4318
 TRACING_ENABLED=1
+
+# redis, not used in dev mode
+#REDIS_ENABLED=true
+#REDIS_PORT=6379
+#REDIS_HOST=localhost
+#REDIS_PASSWORD=vivid
```
[the remark this answers is not in the document]
Comment (nsams) [reply .env:72]: and the other envs (REDIS_PORT/HOST/PASSWORD)? commented out, or not? or remove?

File: docker-compose.yml [hunk 2]
```diff
@@ -42,9 +42,19 @@ services:
...
+#    ports:
+#      - ${REDIS_PORT}:6379
+#    networks:
+#      - redis
+
 networks:
   postgres:
     driver: bridge
+#  redis:
+#    driver: bridge
```
[the remark this answers is not in the document]
Comment (nsams) [reply docker-compose.yml:57]: Please add a suggestion

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**